src/dash-table/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,92 @@
+import * as R from 'ramda';
+import React, { PureComponent } from 'react';
+import Remarkable from 'remarkable';
+
+import { isEqual } from 'core/comparer';
+
+import { MAX_32BITS } from 'dash-table/derived/table/tooltip';
+import { TooltipSyntax } from 'dash-table/tooltips/props';
+
+export enum Arrow {
+    Bottom = 'bottom',
+    Left = 'left',
+    Right = 'right',
+    Top = 'top'
+}
+
+export interface ITooltipProps {
+    arrow?: Arrow;
+    className?: string;
+    tooltip: {
+        delay: number;
+        duration: number;
+        type?: TooltipSyntax;
+        value?: string;
+    };
+}
+
+interface ITooltipState {
+    display?: boolean;
+    displayTooltipId?: any;
+    hideTooltipId?: any;
+    md: Remarkable;
+}
+
+export default class Tooltip extends PureComponent<ITooltipProps, ITooltipState> {
+    constructor(props: ITooltipProps) {
```

Comment:
This is the actual, non-decorated, non-fragment tooltip implementation -- should work everywhere / be reusable.